windows-driver-docs-pr/other-wdk-downloads.md

```diff
@@ -16,32 +16,32 @@ Enterprise WDK (EWDK), and additional downloads for support purposes. To develop
 use the latest public versions of the Windows Driver Kit (WDK) and tools, available for
 download on [Download the Windows Driver Kit (WDK)](./download-the-wdk.md).
 
-To use these earlier versions, you must *first* install the version of
-Visual Studio that is appropriate for your targeted platform.
+To use these earlier versions of WDK, you must *first* install the version of
+Visual Studio and Software Development Kit (SDK) that is appropriate for your targeted platform.
 
 ## Runtime requirements
 
-Starting with the Windows 11, version 22H2 release of the WDK and EWDK, the kits support:
+Starting with the Windows 11, version TBD release of the WDK and EWDK, the kits support:
 
 * Visual Studio 2022 exclusively
 * Building and testing kernel-mode drivers for x64 and Arm64
-* Building and testing drivers for Windows 10, Windows Server 2016 and later client and server versions
+* Building and testing drivers for Windows 11 TBD, Windows Server 2022, and later client and server versions
 * Side by side (SxS) support with previous WDK/EWDK
 
-Multiple WDKs and EWDKs can be installed concurrently on the same computer and even be part of the same build system. You can run the Windows 11, version 22H2 WDK on Windows 7 and later.
+Multiple WDKs and EWDKs can be installed concurrently on the same computer and even be part of the same build system. You can run the Windows 11, version TBD WDK on Windows 7 and later.
 
-To target Windows 8.1, Windows 8, and Windows 7, install an older WDK (Windows 11, version 21H2 and previous) and an older version of Visual Studio either on the same machine or on a separate machine. For links to older kits, see the table below.
+To target Windows 11, Version 23H2, and previous versions of Windows, install a matching version of WDK and Visual Studio either on the same machine or on a separate machine. For links to older kits, see the table below.
 
-Certain device-specific stacks (for example graphics) continue to have x86/ARM32 user-mode components to support x86/ARM32 apps.
+Certain device-specific stacks (for example graphics) continue to have x86 user-mode components to support x86 apps.
 
 Additionally, starting with Windows 11, version 22H2 release of the WDK and EWDK, WDF redistributable co-installers are no longer supported. To learn how to work around this change, see [WDK Known Issues](./wdk-known-issues.md).
 
-You can run the Windows 11, version 21H2 WDK (including the WDK for Windows Server 2022) on Windows 7 and later, to develop drivers for the following operating systems:
+You can run Windows 11, version 23H2 WDK (including the WDK for Windows Server 2022) on Windows 7 and later, to develop drivers for the following operating systems:
```

Comment:
There is no 23H2 WDK.  And this page is all about the previous kits, i.e. not the most current one.